integr8ly/integreatly-operator
# MGDAPI-5000 : Set Outgoing email address

# Issue link

https://issues.redhat.com/browse/MGDAPI-5000

# What

Set outgoing email address as oulined in the following guide. https://access.redhat.com/solutions/4112341

# Verification steps




* Install this branch, pre-built index: quay.io/jfitzpat/managed-api-service-index:1.32.0-5000
* Set up extra tenants after the install, [guide](https://gitlab.cee.redhat.com/rhcloudservices/integreatly-help/-/blob/master/sops/3scale_create_tenants.md)
* Follow the [knowledge article](https://access.redhat.com/solutions/4112341) 
* Check what the current value for the email is.
* Change the current email to something else. 
** The reonciler should revert all email address to the initial values. 
* Make sure to check the UI for the extra tenants
